### PR TITLE
Time server

### PR DESCRIPTION
**Pull request by clayton8**

### CODE REVIEW

File: Sim_Harness_GUI/MainWindow.cs [hunk 1]
```diff
@@ -149,17 +162,58 @@ private bool checkFiles()
 	private string buildStartString()
 	{
 		string jsonString = "{\n\t\"TimeFrame\": {" +
-			"\n\t\t\"wall\": \"" + DateTime.Now.ToString("o") + "\"," +
-			"\n\t\t\"sim\": \"" + DateTime.Now.ToString("o") + "\"," + 
+			"\n\t\t\"wall\": \"" + DateTime.Now.ToString("O") + "\"," +
+			"\n\t\t\"sim\": \"" + _simTime.ToString("O") + "\"," +
 			"\n\t\t\"rate\": " + timeFrameSpeedSpinbutton.Text +
```
Comment (jasedit) [inline Sim_Harness_GUI/MainWindow.cs:167]: You realize this entire statement is solved much more safely by simply using `JsonConvert.SerializeObject()`? I could've sworn I made this change awhile back.

File: Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic [hunk 2]
```diff
@@ -7,22 +7,22 @@
     <widget-library name="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
```
Comment (jasedit) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: This file is a byproduct of the build, and should never be committed. I am fairly confident I had made a PR that deletes this as well.

**Comment by jasedit** [comment]
Ah - 31 commits behind master, and merge conflicts. That would explain exactly why I thought I was going crazy.

File: Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic (hunk 2 above)
Comment (clayton8) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: Sorry I did not realize that these were included. I am going to go fix this merge.

Comment (jasedit) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: Sorry for barging in - I just wanted to make sure my attempts to help weren't lost to the ether. If you need any help, I'm happy to help explain things or offer helpful tricks, just let me know.

Comment (jasedit) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: I can fix this PR for you if you like

Comment (clayton8) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: Hold on don't touch this branch I am making some changes to it. I might be making a new and improved brach to fix some of this code.

Comment (clayton8) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: Give me an hour or so to fix this whole branch haha

Comment (jasedit) [inline Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic:7]: I'm not going to touch it.

**Pull request closed without merging**